Havit/Havit/Screens/Category/View/CategoryCollectionViewCell.swift

```diff
@@ -53,11 +74,24 @@ class CategoryCollectionViewCell: BaseCollectionViewCell {
             $0.bottom.equalToSuperview().inset(19)
         }
 
-        arrowImageView.snp.makeConstraints {
-            $0.top.equalToSuperview().inset(13)
-            $0.trailing.equalToSuperview().inset(11)
-            $0.bottom.equalToSuperview().inset(15)
-            $0.width.height.equalTo(28)
+        switch type {
+        case .category:
+            self.addSubView(arrowImageView)
+
+            arrowImageView.snp.makeConstraints {
+                $0.top.equalToSuperview().inset(13)
+                $0.trailing.equalToSuperview().inset(11)
+                $0.bottom.equalToSuperview().inset(15)
+                $0.width.height.equalTo(28)
+            }
+        case .manage:
+            self.addSubView(editButton)
+
+            editButton.snp.makeConstraints {
+                $0.top.equalToSuperview().inset(14)
+                $0.trailing.equalToSuperview().inset(12)
+                $0.bottom.equalToSuperview().inset(14)
```

Comment:
p2;

```suggestion
                $0.top.bottom.equalToSuperview().inset(14)
                $0.trailing.equalToSuperview().inset(12)
```

이렇게 묶어서 적용해주시면 훨씬 깔끔합니다.

Reply:
@YoonAh-dev  넵 적용 완료 했습니다 !!! 반영커밋 : bdbf198